src/normal.cc

```diff
@@ -1254,7 +1254,7 @@ void select_object(Context& context, NormalParams params)
 
     const int count = params.count <= 0 ? 0 : params.count - 1;
     on_next_key_with_autoinfo(context, KeymapMode::Object,
-                             [count](Key key, Context& context) {
+                             [count,params](Key key, Context& context) {
```

Comment:
Remove the count capture, and declare the count constant directly inside this lambda.

Reply:
done